docs/melange-book/installation.md

```diff
@@ -0,0 +1,59 @@
+# Installation
+
+## Prerequisites
+
+- You are running a Unix system
+  - Windows users are encouraged to install [Windows Subsystem for
+Linux](https://learn.microsoft.com/en-us/windows/wsl/)
+- You have a recent version of, and know how to use:
+  - [Node.js](https://nodejs.org/)
+  - [git](https://git-scm.com/)
+  - [Visual Studio Code](https://code.visualstudio.com/)
+
+## Opam
+
+We need [opam](https://opam.ocaml.org/), the OCaml Package Manager. There are
+many ways to install it depending on your platform, but let's go with the
+simplest method:
+
+    bash -c "sh <(curl -fsSL https://raw.githubusercontent.com/ocaml/opam/master/shell/install.sh)"
+    opam init
+    eval $(opam env)
+
+While `opam init` is running, it will prompt you with something like "Do you
+want opam to modify ~/.profile?". Type `y` to agree.
+
+## Melange opam template
+
+Let's make sure that everything works by downloading and running
+[melange-opam-template](https://github.com/melange-re/melange-opam-template).
+
+    git clone https://github.com/melange-re/melange-opam-template
+    cd melange-opam-template
+    make init
+    make build
+    make serve
+
+While `make init` you can consider grabbing some coffee or other beverage, as it
+might take a while the first time. The last command, `make serve`, should open a
```

Comment:
Probably explaining that the first time: fetches all dependencies and builds them it doesn't need to happen ever again.

Reply:
Added in https://github.com/melange-re/melange-re.github.io/pull/93/commits/047b08c6bf4134904b82aedc4754a4d38949d55e